ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: openshift-manifests-external
+  from: tools
+  commands: openshift-manifests-external-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+  - name: URL
+    default: ""
+  - name: SHA256_HASH
+    default: ""
+  - name: POST_INSTALL
+    default: "false"
```

Comment:
can we add `documentation` to these `env` entries?  Because my impression is that when `POST_INSTALL` is `true`, we are side-loading, and for the default `false`, we are creating manifest files in `SHARED_DIR` to be consumed by a later install step.

Reply:
Good idea, added this in fdee5fdc4fcc8e7226e8a815f5bcd3be9c40a45f